src/services/conversationsService.js

```diff
@@ -409,6 +409,14 @@ const validatePassword = async (password) => {
 	})
 }
 
+const setConversationPicture = async function(token, file) {
+	return axios.post(generateOcsUrl('apps/spreed/api/v1/room/{token}/avatar', { token }), file,)
+}
+
+const deleteConversationPicture = async function(token) {
+	return axios.delete(generateOcsUrl('apps/spreed/api/v1/room/{token}/avatar', { token }))
+}
```

Comment:
In most of places it is called `avatar` in the code. Should we rename to `avatar` here too?